src/ontology/genepio-edit.owl

```diff
@@ -12644,6 +12705,89 @@ AnnotationAssertion(rdfs:label obo:GENEPIO_0100634 "State or Province or Territo
+AnnotationAssertion(obo:IAO_0000119 obo:GENEPIO_0100899 "https://www.britannica.com/science/cecum")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100899 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100899 "2023-11-28T13:29:31Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100899 "caecal content")
+SubClassOf(obo:GENEPIO_0100899 obo:GENEPIO_0100898)
+
+# Class: obo:GENEPIO_0100900 (intestinal content)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100900 "Undigested food material, microbiota, and other materials found in the intestine."@en)
+AnnotationAssertion(obo:IAO_0000119 obo:GENEPIO_0100900 <https://www.britannica.com/science/intestine>)
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100900 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100900 "2023-11-28T13:54:45Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100900 "intestinal content")
+SubClassOf(obo:GENEPIO_0100900 obo:GENEPIO_0100898)
+
+# Class: obo:GENEPIO_0100926 (isolated from single source)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100926 "A process in which specimens are obtained from a single sampling event."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100926 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100926 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100926 <https://orcid.org/0000-0002-1107-9135>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100926 "2023-11-29T11:01:23Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100926 "isolated from single source")
+SubClassOf(obo:GENEPIO_0100926 obo:GENEPIO_0100286)
+
+# Class: obo:GENEPIO_0100929 (PONAR grab sampler)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100929 "A specimen collection device that is used to collect sediment samples from hard bottoms."@en)
+AnnotationAssertion(obo:IAO_0000116 obo:GENEPIO_0100929 "Planned Obsolescence: this term is a placeholder for a term requested in another ontology. Once the appropriate ontology term is available, this term’s identifier will be obsoleted with a “term replaced by” id of the other term.")
+AnnotationAssertion(dcterms:contributor obo:GENEPIO_0100929 <https://orcid.org/0000-0001-5275-8866>)
+AnnotationAssertion(dcterms:date obo:GENEPIO_0100929 "2023-11-29T14:21:43Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:GENEPIO_0100929 "PONAR grab sampler")
+SubClassOf(obo:GENEPIO_0100929 obo:OBI_0002814)
+
+# Class: obo:GENEPIO_0100930 (soil sample probe)
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0100930 "A specimen collection device that is used to collect a soil sample from the ground."@en)
```

Comment:
Perhaps "A specimen collection device designed to penetrate soil with a hollow tube to collect soil sample cores."